docs/firstobject.md

```diff
@@ -76,78 +76,81 @@ There is of course a lot more to Nodes, Content Types, Formatters, Widgets and i
 - The Strawberry field Field Widget will take all your Webform input, process any uploaded files, generate a JSON representation, enrich and complement it with Archipelago specific data and save it for you inside the `Strawberry field`.
 
-We will come back to this later. 
+We will come back to this later.
 
 </div>
 </details>
 
-Make sure you select `Photograph` as `Media Type` and all the fields with a red `*` are filled up. Press `Next: Linked data`. That will load the next step inline. 
-
-Note: Don't press `Save`, Don't press `Preview`. Why? You are not ready yet!
-
 ### Linked Data
 
-![Ingest Step2](../imgs/ingest-step2.jpg)
+As the name of this step suggests; you will be adding all your Linked Data elements here. This step showcases some of the autocomplete Linked Data Webform elements we built for Archipelago. We truly believe in Wikidata as an open, honest, source of Linked Open Data and also one where you can contribute back. But we also have LoC autocompletes and Getty.
 
-As the name of this step suggests; on this page you will be adding all your Linked Data elements. This Step showcases some of the Autocomplete Linked Data Webform elements we built for Archipelago. We truly believe in Wikidata as an open, honest, source of Linked Open Data and also one where you can contribute back. But we also have LoC autocompletes and Getty.
+Again, enter all fields with a red `*` and when you are finished, click `Move on to next step`
 
-Click: `Next: Collections`
+![Ingest Step 2 - Linked Data](../imgs/02_ingest-step-two_linked-data.png)
 
 ### Collections
 
-![Ingest Step3](../imgs/inges-step-3_Collections.jpg)
-
 Since this is our first digital object we do not yet have a Digital Object *Collection* for which `My First Digital Object` could be a member of. In other words, you can leave `Collection Membership` blank and click `Next: Upload Files`.
 
+![Ingest Step 3 - Collections](../imgs/03_ingest-step-three_collections.png)
+
 ### Upload Files
 
-![Ingest Step4](../imgs/ingest-step-4_Upload-files.jpg)
+Now we will upload our `Photograph`. Click `Choose Files` to open your file selector window and choose which file you would like to ingest.
 
-This step is pretty straight forward: Click `Choose Files` to open your file selector window and choose which file you would like to ingest. After you've done this, click `Save Metadata`. 
+![Ingest Step 4 - Attach Files pt 1](../imgs/04_ingest-step-four_upload-files_1.png)
 
-Two things to note: The `Save Metadata` button simply persists the metadata in the current webform session. The actual ingest of the Object happens when you click `Save` (see the **Complete** step below). Secondly, each "upload field" has a set of allowed file types (such as .jpg, .tiff, .png) and in this demo there are also file size restrictions which are explained in the Upload Field's description.
```

Comment:
Is this explanation replaced by something different? I feel we need to explain, until we have a better solutoion, that 'save metadata' is different than the other save.